rotkehlchen/types.py

```diff
@@ -951,6 +951,11 @@ class OptionalChainAddress(NamedTuple):
     blockchain: SupportedBlockchain | None
 
 
+class OptionalBlockchainAddress(NamedTuple):
```

Comment:
Can't this just be the same as above with a Typevar for the type of address? No need to create another tuple